client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManagerList.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Checkbox,
+  Divider,
+  DropDownMenu,
+  List,
+  ListItem,
+  MenuItem,
+  Subheader,
+} from 'material-ui';
+import { grey400 } from 'material-ui/styles/colors';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { memberShape } from '../../../propTypes';
+
+const translations = defineMessages({
+  normal: {
+    id: 'course.group.show.groupUserManagerList.normal',
+    defaultMessage: 'Normal',
```

Comment:
Should we call this member? Initially I thought this would be for 'normal' (vs phantom) student. Also, can we see if a certain user is phantom?

Reply:
Changed to member. And yeah I think can add the phantom indicator, will do so